server/src/main/java/org/elasticsearch/script/ScriptCacheStats.java

```diff
@@ -121,13 +121,6 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
         return context;
     }
 
-    /**
-     * Get the general stats, null if using context cache
-     */
-    public ScriptStats getGeneralStats() {
-        return general;
```

Comment:
Please also remove the member variable for this as well.

Reply:
Removed.